### PR TITLE
adding domains field to account

### PR DESCRIPTION
**Pull request by kpsf**
added domains account field to metadata and permissions

### CODE REVIEW

**Comment by wyattsweet** [comment]
The field is not getting created on my dev org. Here’s what I did –
1. pulled and am working off the issue branch
2. rebuilt the project
3. deployed all metadata
3. logged in to my dev org connected to my test Desk site
4. Once connected checked the object manager, accounts and didn't see the field

@tstachl any ideas on something I might be doing wrong above?

File: src/package.xml [hunk 1]
```diff
@@ -70,6 +71,7 @@
         <members>Contact.HomeAddress__c</members>
         <members>Contact.Language__c</members>
         <members>Contact.WorkAddress__c</members>
+        <members>Contact.Background__c</members>
```
Comment (tstachl) [inline src/package.xml]: @kpsf this should not be part of this pull request.

Comment (kpsf) [inline src/package.xml]: Hmm, yeah that was from an old local change. How do I remove?

Comment (wyattsweet) [inline src/package.xml]: I went to object manager > account and wasn't able to find the field

screen shot 2017-03-07 at 4 54 45 pm

Comment (kpsf) [inline src/package.xml]: Fixed

**Comment by wyattsweet** [comment]
I tested this issue using the following steps: 

1. Pulled down the latest changes and the working branch from Github

2. Rebuilt the project in my IDE and **force saved** the `Account.object` and recreated the permission set by force saving `SCMT_Audit.permissionset`

3. Confirm the custom field was created and added to the correct permission set

screen shot 2017-03-08 at 11 15 52 am

screen shot 2017-03-08 at 11 16 58 am

**Comment by kpsf** [comment]
pls approve
